css/style.css

```diff
@@ -0,0 +1,81 @@
+body{
+	text-align: center;
+}
+#board{
+	margin: 0 auto;
+	width: 314px;
+	height: 314px;
+	background-color: black;
+	display: inline-block;
+	
+}
+
+.boardSpace{
+	border: 1px solid white;
+	width: 100px;
+	height: 100px;
+	display: inline-block;
+	margin: 0 auto;
+	background-color: white;
+}
+
+.x{
+	background-image: url("../images/x.png");
+	background-size: cover;
+}
+
+.o{
+	background-image: url("../images/o.png");
+	background-size: cover;
+}
+
+#alert, #winner{
+	position: fixed;
+	margin: 0 auto;
+	height: 160px;
+	width: 300px;
+	margin-top: 80px;
+	margin-left: 5px;
+	text-align: center;
+	background-color: white;
+	border: 2px solid black;
+	display: none;
+}
+
+#alert h1, #winner h1{
+	margin-top: 51px;
+}
+
+#exit{
+	position: absolute;
+	top: 0;
+	right: 0;
+	background-image: url("../images/x.png");
+	width: 10px;
+	height: 10px;
+}
+
+#button{
+	display: block;
+	margin: 0 auto;
+	margin-top: 20px;
+}
+
+#turnX, #turnO{
+	height: 100px;
+	width: 100px;
+	margin-left: 50px;
+	margin-right: 50px;
+	display: inline-block;
+	vertical-align: top;
+	background-size: cover;
+}
+
+#turnX{
+	background-image: url("../images/X.png");
+}
+
+#turnX p, #turnO p{
+	margin: 0 auto;
+
+}
```

Comment:
Nice job with the CSS. Looks like you made a lot of adjustments, and it resulted in a clean, simple page that looks good as you resize the page.